packages/react-composites/src/composites/CallComposite/pages/ConfigurationPage.tsx

```diff
@@ -108,11 +112,50 @@ export const ConfigurationPage = (props: ConfigurationPageProps): JSX.Element =>
   /* @conditional-compile-remove(rooms) */
   mobileWithPreview = mobileWithPreview && rolePermissions.cameraButton;
 
+  /* @conditional-compile-remove(call-readiness) */
+  const [isDrawerShowing, setIsDrawerShowing] = useState(true);
+  /* @conditional-compile-remove(call-readiness) */
+  const onLightDismissTriggered = (): void => {
+    // do nothing here
+    // only way to dismiss this drawer is clicking on allow access which will leads to device permission prompt
+  };
+  /* @conditional-compile-remove(call-readiness) */
+  const drawerStyle: _DrawerSurfaceStyles = {
+    root: {
+      position: 'absolute',
+      top: 0,
+      left: 0,
+      width: '100%',
+      height: '100%',
+      // apply zindex = 99 so drawer appear over device buttons and other components in the config page
+      zIndex: 99
```

Comment:
should we maybe have this just 1 above the highest layer in the composites config page? avoid this magic number?

Reply:
Can we use common or existing styles, drawer is shown in other places:
![image](https://user-images.githubusercontent.com/2684369/197644052-af82f098-b453-4229-a349-dc20d8d2d802.png)

Reply:
We can't coz the buttons are set as zindex: 11

Reply:
could we make another PR to bring that z-index down if it dosen't break anything? would that be considered a breaking change @JamesBurnside since some Contosos might have styled their apps around that?

Reply:
They should be set to 11 for a reason - every z-index should be something + 1 instead of arbitrary values

Reply:
We should still aim to use common styles though, if someone needs to add some new css property to all uses of the drawer they'll need to add it in x different places and could easily miss one.
Consider 
```ts
/**
  Drawer styles to be used to house the _DrawerComponent on top of all other content on the screen.
  @param zIndex: this defaults to MAX_Z_INDEX if unset
*/
const drawerContainerStyles = (zIndex: number = MAX_Z_INDEX): IStackStyles => ({
  root: {
   ...etc.
  }
});
```

Reply:
fixed